sycl/include/sycl/ext/oneapi/device_global/device_global.hpp

```diff
@@ -153,7 +161,7 @@ class
     return *this->get_ptr();
   }
 
-  const T &get() const noexcept {
+  constexpr const T &get() const noexcept {
```

Comment:
Strictly speaking, this changes the definition from the [extension specification](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc). If we need this, we should also change the docs.